tests/network-tests/src/Api.ts

```diff
@@ -1918,4 +1925,132 @@ export class Api {
   public getMaxWorkersCount(module: WorkingGroups): BN {
     return this.api.createType('u32', this.api.consts[module].maxWorkerNumberLimit)
   }
+
+  async sendContentDirectoryTransaction(operations: OperationType[], pair: KeyringPair): Promise<void> {
```

Comment:
since this is sending transactions in the context of the "Lead", you should not need to pass the KeyringPair all the way from the scenario to through the flows to this api call.

You can simplify here by getting the current content directory lead, and get their role account address and use in the call to sender.signAndSend

Reply:
Done here https://github.com/Joystream/joystream/pull/1565/commits/defd9c5f5013d5ea5060efedbe0dfc8b28bc917f